utensor_cgen/utils.py

```diff
@@ -118,18 +118,25 @@ def parse_tensor_name(tname):
 
 class NamescopedKWArgsParser:
 
-  def __init__(self, name_space, kwargs):
+  def __init__(self, name_space, data_manager, op_info, kwargs):
```

Comment:
For backward compatibility, it's better to make `data_manager` and `op_info` keyword arguments.
Or you'll fix all codes where use `NamescopedKWArgsParser`.